Lib/idlelib/util.py

```diff
@@ -13,8 +13,19 @@
     * warning stuff (pyshell, run).
 """
 
+import os
+
+
 # .pyw is for Windows; .pyi is for stub files.
-py_extensions = ('.py', '.pyw', '.pyi')  # Order needed for open/save dialogs.
+py_extensions = {'.py', '.pyw', '.pyi'}  # Order needed for open/save dialogs.
```

Comment:
I didn't have to change this, but a _set_ is a more correct data type for this application.

Reply:
Someone could argue that a tuple is a more useful type, as `py` is much more used than the others!

Reply:
Yes, in the initial version of my PR that added the `idlelib.util` submodule, I used a `frozenset` here. But @terryjreedy pointed out that the order of these extensions was important, and that therefore a `frozenset` would be unsuitable.

Reply:
No sweat. I'll revert the last commit.

Reply:
Resolved with 0c6e0687391744f1b51e5dff32343d97583fc032